src/lv_fonts/lv_font_template.c

```diff
@@ -113,7 +113,7 @@ static uint8_t kern_right_class_mapping[] =
 /*Kern values between classes*/
 static uint8_t kern_class_values =
 {
-        32, 33, 22, 11, 55, 33, 22, 44
+        LV_FONT_X4_SET(2, 7), LV_FONT_X4_SET(-1, 3), LV_FONT_X4_SET(0, 13), LV_FONT_X4_SET(5, 7)
```

Comment:
`LV_FONT_X4_SET(-1, 3)` seems all negatives will have wrong fractional part with current implementation.

I'd suggest to use integers (real value * 16) + leave inline comment `/* FP12.4 */`. It's not so difficult to divide if needed.

Reply:
Or even better - post real float value in inline comment. Much more clear than macros magick

Reply:
> Or even better - post real float value in inline comment. Much more clear than macros magick

`floats` are "forbidden" in LittelvGL to keep size simple. Using one float number will require to add the whole float library.

Anyway, I added comments instead.